packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -594,4 +594,37 @@ describe("calcite-alert", () => {
   describe("translation support", () => {
     t9n("calcite-alert");
   });
+
+  describe("theme", () => {
+    themed(
+      html`
+        <calcite-alert label="this is a default alert" scale="s">
+          <div slot="${SLOTS.title}">Test title</div>
+          <div slot="${SLOTS.message}">Test message</div>
+        </calcite-alert>
+      `,
+      {
+        /*"--calcite-alert-width": {
+        shadowSelector: `.${CSS.container}`,
```

Comment:
if you search for `--calcite-alert-width` in the SCSS you will see it only applies to `:host` so you can remove the `shadowSelector` here. Either explicitly set `selector: "calcite-alert"` or take it off completely.